operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -191,11 +192,9 @@ private static void begin() {
 
       Step strategy = Step.chain(
           new InitializeNamespacesSecurityStep(targetNamespaces),
-          new NamespaceRulesReviewStep(),
-          CrdHelper.createDomainCrdStep(version,
-              new StartNamespacesStep(targetNamespaces, false)));
+          new NamespaceRulesReviewStep());
       if (!isDedicated()) {
-        strategy = Step.chain(strategy, readExistingNamespaces());
+        strategy = Step.chain(strategy, readExistingNamespaces(targetNamespaces));
```

Comment:
If I understand the change correctly, domain CRD creation and  start namespaces logic are associated with readExistingNamespaces(), which is skipped in the dedicated mode. Does that mean in dedicated mode, CRD and domains will not be created? I may have missed something. Have you tested dedicated mode use cases?

Reply:
Thanks Dongbo. I'm trying to run integration tests in external jenkins with my changes and seeing some failures. I'm not sure if failures are related to my change or intermittent env issue. I'm also trying to run tests in my machine but seeing a different issue regarding JAVA_HOME.  Do you know if there are any integration test for dedicated mode?

Reply:
Hi Dongbo, I have adjusted the change for dedicated mode as we discussed. Please let me know if you see something missing. I'll also try running tests for dedicated mode use-case. Thanks again for your comment and catching this.

Reply:
The modified code here looks good.